fuzzing/src/utils/utils.h

```diff
@@ -28,18 +28,18 @@
 
 #include <memory>
 
-#include <archive.h>
-
 #include <ft2build.h>
 #include FT_FREETYPE_H
 #include FT_GLYPH_H
 
 
+struct archive;
+
```

Comment:
not sure exactly about this case but let's maybe also add a double empty line after the declaration

Reply:
Done.